public/js/reducers/chartsReducer.js

```diff
@@ -97,6 +100,34 @@ const charts = (state = initialState, action) => {
     case 'GET_IN_WORKFLOW_VS_NOT_IN_WORKFLOW_FAILED':
         return Object.assign({}, state, { inWorkflowVsNotInWorkflow: { ...state.inWorkflowVsNotInWorkflow, error }});
         
+    case 'UPDATE_FORK_TIME': {
+        const forkTimeData = chartData.data.sort(compareIssueDates);
+        const forkTimeSeries = [{ data: forkTimeData }];
+        const seriesWithLabels = forkTimeSeries.map(series => {
+            return {
+                data: series.data.map((dataPoint) => {
+                    const date = moment(dataPoint['issueDate']).utc();
+                    return {
+                        x: date.valueOf(),
+                        y: dataPoint.secondsUntilFork,
+                        label: `${(dataPoint.secondsUntilFork / 3600).toFixed(1)} hours`,
+                        size: 2.5
```

Comment:
Why 2.5?

Reply:
this is the radius of a single dot: I set it to 2.5 which makes dots smaller than the default value, but still big enough